### ISSUE TITLE
ansible-lint 25.2.0 and above does not work with every rules and neovim integration

### ISSUE THREAD

**Issue by ginolegigot** [ansible/ansible-lint#4715]
##### Summary

i cant find logs so i'm opening this. i use neovim (0.11.3 so the latest stable version) and the ansible-language-server. I found some errors dont show up since 25.2.0.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
❯ uv run ansible-lint --version
ansible-lint 25.2.0 using ansible-core:2.18.7 ansible-compat:25.6.0 ruamel-yaml:0.18.14 ruamel-yaml-clib:0.2.12
```

uv add ansible
uv add ansible-lint==25.2.0

##### STEPS TO REPRODUCE

- uv init --python 3.12
- uv add ansible
- uv add anisble-lint==25.1.3
- uv run neovim playbook.yml

A working snippet to test:
```yaml
---
- name: Installing and configuring an airflow instance
  hosts: myhosts
  gather_facts: true
  any_errors_fatal: true
  tasks:

    - name: mount Disks
      ansible.builtin.debug:
        var: ansible_facts
      ignore_errors: true
```
in this snippet two errors are supposed to show:
- ignore errors error which shows up in both versions of ansible-lint
- name[casing] which is not showed in 25.2.0 but it is properly showed in 25.1.3

I also tried to disable validation from the language server and use nvim-lint instead, i reproduce the same issue, that is why i thought it was an ansible-lint issue and not an ansible-language-server issue, nvim-lint being quite minimalist.

However, I also tried vscode and can't reproduce the issue, everything works fine in 25.1.3 and 25.2.0 so i don't know.

If i'm not in the proper place could you indicate me somewhere where i could report the issue ?
Thanks for your time.

**Comment by alisonlhart** [comment]
A few things could be happening here. We have not released the ansible-language-server independently since it was merged into the vscode-ansible project. We have plans to resume these releases in the near future. A new release of the package could resolve the issue, but this is not determined yet. 

We also do not manage nvim-ansible or nvim-lint. 

I understand the issue is originating in `25.2.0`, have you tried the most recent release of lint as well?

**Comment by ginolegigot** [comment]
The bug vanished without any changes on my side, i close it. Thanks for your answer
